test/types/where.ts

```diff
@@ -144,27 +144,27 @@ MyModel.findAll({
   where: {
     id: {
       [Op.in]: {
-        // @ts-expect-error - cannot use Operator inside another one!
+        // @ts-expect-error -- cannot use Operator inside another one!
         [Op.eq]: [1, 2],
       },
     },
   },
 });
 
-// @ts-expect-error - no attribute
+// @ts-expect-error -- no attribute
 MyModel.findAll({
   where: [1, 2],
 });
 
 // TODO [2022-05-26]: TS < 4.4 does not detect an error here. Uncomment test once we remove support for TS 4.3
```

Comment:
Not directly related to this PR, but I believe these tests are uncommented now and this TODO can be removed, also holds for a few other instances in this file

Reply:
Done